codejail/tests/test_jail_code.py

```diff
@@ -161,7 +161,7 @@ def test_directories_are_copied(self):
         self.assertResultOk(res)
         self.assertEqual(res.stdout, textwrap.dedent("""\
             ('.', ['pylib', 'tmp'], ['hello.txt', 'jailed_code'])
-            ('./pylib', [], ['module.py', 'module.pyc'])
+            ('./pylib', [], ['module.py'])
```

Comment:
I don't understand why the pyc is removed from the expected results?

Reply:
I don't know why we would expect it to be there. It's not checked into the repo (https://github.com/edx/codejail/tree/edx-platform-master/codejail/tests/pylib) and the `module.py` file isn't read by the interpreter, as it is copied to `/tmp` and the jailed code doesn't read it.